metadata.md

```diff
@@ -0,0 +1,222 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.9.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+```{currentmodule} tskit
+```
+
+
+(sec_tutorial_metadata)=
+
+# Working with Metadata
+
+Metadata is information associated with entities that tskit doesn't use or interpret,
+but which is useful to pass on to downstream analysis such as sample ids, dates etc. (see
+{ref}`sec_metadata` for a full discussion). Each {ref}`table<sec_tables_api_tables>` has a
+{class}`MetadataSchema` which details the
+contents and encoding of the metadata for each row. A metadata schema is a JSON document
+that conforms to [JSON Schema](https://json-schema.org/understanding-json-schema/)
+(The full schema for tskit is at {ref}`sec_metadata_schema_schema`). Here we use an
+existing tree sequence which contains some demonstration metadata:
+
+```{code-cell} ipython3
+import tskit
+import json
+
```

Comment:
nice to have something telling us where this ts came from. Could include the commented code in a hidden cell or something?

Reply:
Good point. Will do